packaging/flavorgen/flavors/generators.go

```diff
@@ -338,9 +338,6 @@ func newMachineDeployment(cluster clusterv1.Cluster, machineTemplate infrav1.VSp
 		Spec: clusterv1.MachineDeploymentSpec{
 			ClusterName: clusterNameVar,
 			Replicas:    pointer.Int32Ptr(int32(555)),
-			Selector: metav1.LabelSelector{
```

Comment:
since you're updating here. can you add `UseExperimentalRetryJoin` to the kubeadmConfig spec ?

Reply:
good shout